msal/tests/Test.MSAL.NET.UIAutomation/AppFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+
+namespace Test.MSAL.NET.UIAutomation
+{
+    /// <summary>
+    /// Initializes the app object that represents the main gateway to interact with the app on the device
+    /// </summary>
+	public class AppFactory
```

Comment:
>AppFactory [](start = 14, length = 10)

any way this can be shared between ADAL and MSAL ... e.g. by a shared test core?

Reply:
Yes, I can do that